source/vir/src/triggers_auto.rs

```diff
@@ -41,13 +43,15 @@ type Terms = Arc<Vec<Term>>;
 #[derive(PartialEq, Eq, Hash)]
 enum TermX {
     Var(UniqueIdent),
+    VarAt(Ident, VarAt),
```

Comment:
Rather than adding a new TermX case, it would be less intrusive to make this a case in App, just like App::Const.  This way, things like structure_matches and remove_obvious_potential_loops don't need any extra changes.